src/http/client.rs

```diff
@@ -206,6 +206,17 @@ pub struct Http {
     application_id: AtomicU64,
 }
 
+macro_rules! api {
+    ($self:ident, $string:literal $($args:tt)*) => {
+        format!(concat!("{}", $string), $self.api $($args)*)
+    };
+}
+macro_rules! status {
+    ($self:ident, $string:literal $($args:tt)*) => {
+        format!(concat!("{}", $string), $self.status_api $($args)*)
+    };
+}
+
```

Comment:
I think hardcoding the use of `self` here is better than passing it in every time as the first argument.

Reply:
That is not possible due to macro hygiene rules